abc/encoding-a.clp

```diff
@@ -4,7 +4,7 @@
 #include "./action-MPP.clp".
 
 % Order fulfillment (Goal condition)
-#include "./goal-D-a.Clp".
+#include "./goal-D-a.clp".
```

Comment:
the extension indicates a semantic difference

Reply:
But there was no such file...

Reply:
I am checking this problem, there is something strange with the files.